lib/generator/files/generator_cases.rb

```diff
@@ -3,7 +3,9 @@ module Files
     module GeneratorCases
       class << self
         def available(track_path)
-          cases_filepaths(track_path).map { |filepath| slugify(filepath) }.sort
```

Comment:
I strongly disagree with this change. 

It is true that available does not need to know exactly how a test case is generated, and you could achieve this by abstracting out the availability criteria, but since that is all this class does that seems excessive.

> because that might go away or get changed. 

This is a case of YAGNI and we can worry about that when it happens.

See also: https://github.com/exercism/xruby/pull/646 which aims to give new generator implementers helpful advice.